packages/dynamite/dynamite_runtime/lib/src/dynamite_client.dart

```diff
@@ -76,7 +76,7 @@ class DynamiteRawResponse<B, H> {
         _rawBody = switch (bodyType) {
           const FullType(Uint8List) => await response.bytes,
           const FullType(String) => await response.string,
```

Comment:
Please also rename those to `bytesBody` and `stringBody`

Reply:
I think we should rather drop the `Body` suffix from xml and json.
The extension methods are on `Stream<List<int>>` and therefore have nothing to do with the response itself.

What do you think?

Reply:
Sounds good